### ISSUE TITLE
Issue creation as a contribution

### ISSUE THREAD

**Issue by jeaye** [AurelienLourot/github-contribs#1]
The Github profile page also takes into account issue creation as part of the contributions, but it seems github-contribs does not. Is there any reason why this wasn't included? Can it be added?

**Comment by AurelienLourot** [comment]
Hi @jeaye :)

> Is there any reason why this wasn't included?

No special reason, I just didn't need it for https://github.com/AurelienLourot/ghuser.io and I wanted to avoid unnecessary HTTP GETs because of the rate limit.

> Can it be added?

Yes :) I'll try to do it today. I'll probably make it a `--flag` and not change the default behavior.

**Comment by jeaye** [comment]
Sweet! Thanks for the quick and helpful reply. An optional flag for this sounds good to me. :)

**Comment by AurelienLourot** [comment]
Bear with me please :) I have implemented the whole thing but at the last moment I discovered that it wasn't very reliable. So I dug a bit and this is what happens: https://github.com/AurelienLourot/github-contribs/commit/bd745c85eb9c29f622f20775bbfbeaf8ab674c93#diff-04c6e90faac2675aa89e2176d2eec7d8R72

TL;DR "hot issues" need to be fetched differently from "non-hot" ones.

I have now also written a failing unit test for this and I need to implement this part as well. I hope I won't hit anything else.

**Comment by AurelienLourot** [comment]
done :)

```bash
$ sudo npm install -g @ghuser/github-contribs@2.2.0
$ github-contribs --verbose --issues --since 2018-08-08 jeaye
✔ Fetched first day at GitHub: 2011-09-17.
⚠ Be patient. The whole process might take up to a few seconds...
2018-08-09: (hot issues) zulip/docker-zulip
2018-08-08: (hot issues) AurelienLourot/github-contribs
✔ Fetched all commits, PRs and issues.
2 repo(s) found:
zulip/docker-zulip
AurelienLourot/github-contribs
```

**Comment by jeaye** [comment]
Awesome job! This worked very well for me. Thank you! :)

**Comment by AurelienLourot** [comment]
Thank you :)
